components/movementsensor/rtkutils/ntrip.go

```diff
@@ -235,19 +237,15 @@ func (n *NtripInfo) Connect(ctx context.Context, logger logging.Logger) error {
 		}
 
 		c, err = ntrip.NewClient(n.URL, ntrip.Options{Username: n.username, Password: n.password})
```

Comment:
If you want to keep the second sentence, maybe add a comment here to mention why we do not declare the client here as well?